api/api.go

```diff
@@ -210,10 +210,16 @@ type AuthProvider interface {
 	LogoutHandler() http.HandlerFunc
 }
 
+// TODO implement
+
+// IconDescriber optionally provides an icon
+type IconDescriber interface {
```

Comment:
Was macht das? Würde es nicht reichen das Icon am Fahrzeug (Gerät) zu definieren? Oder ist das beim Heizstab immer ein "unbekanntes Fahrzeug" und daher der Wunsch das am Charger schon festzulegen?

Reply:
> Was hältst Du davon, dafür auch ein Icon am Charger definierbar zu machen? Spannend wird nochmal, wie ein Vehicle ohne Namen (da ohne Vehicle) dann eigentlich aussehen soll... Auch "verbunden" als Status muss entfallen.

Sorry, den Kommentar hab ich erst danach gelesen. Ignoriere das was ich geschrieben habe. Ja, vermutlich ist es Sinnvoll die "Zwischenebene Fahrzeug" dann Weg zu lassen. Den Status sollten wir weiterhin beibehalten. Zum Aktivieren/Deaktivieren ist der ja relevant, vermutlich muss man da dann eine bessere Formulierung finden.

Reply:
> Was hältst Du davon, dafür auch ein Icon am Charger definierbar zu machen?

Ja, aber der hätte nur einen Effekt im `fixed` Szenario. Ich würd im Normalfall keine zwei Icons (Loadpoint + Vehicle) sehen.

Reply:
> Was macht das? Würde es nicht reichen das Icon am Fahrzeug (Gerät) zu definieren? Oder ist das beim Heizstab immer ein "unbekanntes Fahrzeug" und daher der Wunsch das am Charger schon festzulegen?

Ja genau. In dem Fall würde sich ein Fahrzeug ja sehr komisch anfühlen.